src/Common/Constants.ts

```diff
@@ -240,4 +240,6 @@ export const RepositoryAction = {
     CONTAINER: "CONTAINER",
     CHART_PULL: "CHART_PULL",
     CHART_PUSH: "CHART_PUSH"
-}
\ No newline at end of file
+}
+
+export const MAX_Z_INDEX = 2147483647
```

Comment:
Confirm the same on all supported browsers

Reply:
It is except on safari=3, even there if we put more than their required it will by default cap it to max.